src/ui/handler/drag_rotate.js

```diff
@@ -258,6 +270,8 @@ class DragRotateHandler {
     _unbind() {
         window.document.removeEventListener('mousemove', this._onMouseMove, {capture: true});
         window.document.removeEventListener('mouseup', this._onMouseUp);
+        window.document.removeEventListener('touchmove', this._onMouseMove, { capture: true });
```

Comment:
should be followed the coding convention `{ capture: true }` to  `{capture: true}` as line no.271

Reply:
Didn't change coding convention it already had..

Reply:
I see, there's already used in mixed way.

Reply:
However I fixed that ;)